byron/semantics/executable-spec/src/Control/State/Transition/Generator.hs

```diff
@@ -67,20 +69,16 @@ import qualified Hedgehog.Gen as Gen
 import           Hedgehog.Range (Size (Size))
 import qualified Hedgehog.Range as Range
 
---------------------------------------------------------------------------------
--- Temporary imports till hedgehog exposes interleaveTreeT and withGenT
---------------------------------------------------------------------------------
-import           Hedgehog.Internal.Gen
-import           Hedgehog.Internal.Tree
---------------------------------------------------------------------------------
--- END: Temporary imports till hedgehog exposes interleaveTreeT and withGenT
---------------------------------------------------------------------------------
+import           Hedgehog.Internal.Gen (integral_, runDiscardEffectT)
```

Comment:
Are these no longer temporary? We're still relying on "internal" things?

Reply:
Right, but in this case we don't require the unexposed `interleaveTreeT` and `withGenT`, so there's no need to re-implement them here. But we still need the internal functions of `hedgehog` and this is not temporary :) 

I added the `temporary` comments to signal that we could remove them once submitting a PR for exposing the two functions above, but we don't need these anymore ...